docs/Users_Guide/quicksearch.rst

```diff
@@ -12,29 +12,31 @@ METplus Quick Search for Use Cases
 Use Cases by MET Tool:
 ======================
 
+.. |branch| replace:: feature_963_quick_search
```

Comment:
This line can be removed since it is no longer needed.

Reply:
@georgemccabe Great catch!  Sorry I missed that.  Thanks!  I just removed that line in the feature branch.